src/ngraph/op/cum_sum.hpp

```diff
@@ -0,0 +1,99 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/axis_set.hpp"
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        // clang-format off
+       /// \brief Tensor cumulative sum operation.
+       ///
+       /// Compute the cumulative sum of the input tensor along the axis specified.
+       ///
+       /// ## Parameters
+       ///
+       /// |                      | Description                              							   |
+       /// | -------------------- | --------------------------------------------------------------------------------------------------|		
+       /// | `exclusive`          | If set to 1 will return exclusive sum in which the top element is not included. 		   |
+       ///	|		       | In other terms, if set to 1, the j-th output element would be the sum of the first (j-1) elements.|. 
+       ///	|		       | Otherwise, it would be the sum of the first j elements.  					   |
+       ///
+       /// |                      | Description                                        |
+       /// | -------------------- | -------------------------------------------------- |
+       /// | `reverse`            | if set to 1, performs the sum in reverse direction |
+       ///
+       /// ## Inputs
+       ///
+       /// |       | Description                                            |
+       /// | ----- | ------------------------------------------------------ |
+       /// | `arg` | An input tensor of any shape and numeric element type. |
+       ///
+       /// |       | Description                                                     |
+       /// | ----- | --------------------------------------------------------------- |
+       /// | `axis`| An axis position along which cumulative sum must be performed.  |
+       ///
+       /// ## Output
+       ///
+       /// | Description                                                                         |
+       /// | ------------------------------------------------------------------------------------|
+       /// | Output tensor of the same type as `arg` with cumulative sums of the arg's elements  |
+       //
+       // clang-format off
```

Comment:
I suggest to `// clang-format on` here and apply style for the rest of code in this file. 
There is some indentation issues.